Packs/Digital_Defense_FrontlineVM/Playbooks/playbook-Digital_Defense_FrontlineVM_-_PAN-OS_block_assets.yml

```diff
@@ -247,7 +247,9 @@ tasks:
       id: 963529eb-2f34-4072-801f-9eed161669d1
       version: -1
       name: PAN-OS - Block IP and URL - External Dynamic List
-      description: ''
+      description: |-
+        This playbook blocks IP addresses and URLs using Palo Alto Networks Panorama or Firewall External Dynamic Lists.
```

Comment:
where did you take this from

Reply:
when I downloaded it from demisto - it added this description. should I take it out?

Reply:
Make sure with @idovandijk @bakatzir